src/test/java/com/sfuentes/SupermarketTest.java

```diff
@@ -0,0 +1,38 @@
+package com.sfuentes;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class SupermarketTest {
```

Comment:
I have more test cases in mind, for example:
What happens if I add the same item name twice? will that give me the correct amount?
What happens if I add a null key? or a null price?
etc.
Try to come up with all the possible test cases.

Reply:
fixed